### PR TITLE
Extend .mailmap to eliminate some duplicates in `git shortlog -se`

### PR DESCRIPTION
**Pull request by JasonGross**
If there was already an account in .mailmap, I preferred the one already present.  If there was not, I preferred whichever account had more commits.

Additions I'm unsure about:
- [x] @chdoc you have 1 commit under christian.doczkal@ens-lyon.fr and 1 commit under christian.doczkal@inria.fr; do you have a preference for how you show up in `git shortlog -nse`?
- [x] @k4rtik you have 1 commit under ks@cs.uchicago.edu and 1 commit under kartiksinghal@gmail.com; do you have a preference for how you show up in `git shortlog -nse`?
- [x] @Janno do you want your email address in `git shortlog -nse` to be Janno@users.noreply.github.com or janno@bedrocksystems.com or janno@mpi-sws.org?
- [x] @WojciechKarpiel you have 1 commit under w.karpiel@o2.pl which is already in .mailmap, and 8 commits under w.karpiel@avsystem.com; any preferences for how you show up in `git shortlog -nse`?

### CODE REVIEW

**Comment by WojciechKarpiel** [comment]
I prefer `w.karpiel@o2.pl`, just the way it is in the PR right now (I've unintentionally used the other one)

Thank you for asking :)

**Comment by chdoc** [comment]
Both addresses are from previous affiliations and essentially defunct. The Inria one has has a permanent forward, so maybe use that.

**Comment by k4rtik** [comment]
> * 1 commit under [ks@cs.uchicago.edu](mailto:ks@cs.uchicago.edu) and 1 commit under [kartiksinghal@gmail.com](mailto:kartiksinghal@gmail.com); do you have a preference for how you show up in `git shortlog -nse`?

Gmail will be better for me long term. Thanks for asking.

**Comment by gmalecha** [comment]
Is it necessary to unify email addresses? In particular, regarding personal vs company commits. I wouldn't expect him to use bedrocksystems for everything, but using it for contributions made through bedrock is preferable.

**Comment by Zimmi48** [comment]
No one is forcing anyone to use a single email address for all of their commits. We are just talking about counting individual committers here (not counting a single individual multiple times just because they have changed employers). That's the purpose of `.mailmap` (allow merging identities without forcing people to use a single address). Also note that you have to use specific options to use the mailmap identity merging feature. If you do a plain `git log`, you will see for each commit which actual email address they have used.

**Comment by Janno** [comment]
Let's go with `janno@mpi-sws.org`. Given my current progress on finishing my PhD that one will stay alive for a long long time even without a redirect..

**Comment by JasonGross** [comment]
This is ready for review and merge.  Can I get an assignee?

File: .mailmap [hunk 1]
```diff
@@ -54,13 +55,16 @@ Maxime Dénès <mail@maximedenes.fr>                 Maxime Dénès <maxime.dene
...
+Christian Doczkal <christian.doczkal@inria.fr>     Christian Doczkal <christian.doczkal@ens-lyon.fr>
 Damien Doligez <doligez@gforge>                    doligez <doligez@85f007b7-540e-0410-9357-904b9bb8a0f7>
 İsmail Dönmez <ismail-s@users.noreply.github.com>  Ismail <ismail-s@users.noreply.github.com>
 formalize.eth <formalize@protonmail.com>           ilya <ilya@localhost.localdomain>
 Andres Erbsen <andreser@mit.edu>                   Andres Erbsen <andres@kevix.co>
 Jim Fehrle <jim.fehrle@gmail.com>                  Jim <jfehrle@sbcglobal.net>
 Jim Fehrle <jim.fehrle@gmail.com>                  Jim Fehrle <jim.fehrle@gmail.com>
 Jim Fehrle <jim.fehrle@gmail.com>                  jfehrle <jfehrle@users.noreply.github.com>
+Jim Fehrle <jim.fehrle@gmail.com>                  Jim Fehrle <jfehrle@sbcglobal.net>
+Jim Fehrle <jim.fehrle@gmail.com>                  Jim Fehrle <jim.fehrle@gmail.com>
```
Comment (silene) [inline .mailmap:67]: What is happening here? Isn't it the identity? And wasn't it already there above?

File: .mailmap [hunk 2]
```diff
@@ -86,6 +90,7 @@ Jason Gross <jgross@mit.edu>                       JasonGross <JasonGross@users.
 Vincent Gross <vgross@gforge>                      vgross <vgross@85f007b7-540e-0410-9357-904b9bb8a0f7>
 Samuel Gruetter <samuel.gruetter@epfl.ch>          Samuel Gruetter <saemi472@gmail.com>
 Huang Guan-Shieng <huang@gforge>                   huang <huang@85f007b7-540e-0410-9357-904b9bb8a0f7>
+Armaël Guéneau <armael.gueneau@ens-lyon.fr>        Armael <armael@isomorphis.me>
```
Comment (silene) [inline .mailmap:93]: He prefers we go with the following address:
```suggestion
Armaël Guéneau <armael.gueneau@ens-lyon.org>       Armael <armael@isomorphis.me>
Armaël Guéneau <armael.gueneau@ens-lyon.org>       Armaël Guéneau <armael.gueneau@ens-lyon.fr>
```

**Comment by Alizter** [comment]
@JasonGross I'm happy to merge this soon. Can you address silene's comment above and perhaps do a rebase?

File: .mailmap (hunk 1 above)
Comment (JasonGross) [reply .mailmap]: Hm, yeah, not sure.  I've removed the duplicate

**Comment by JasonGross** [comment]
Should be ready for review and merge, sorry for the delay

**Comment by Alizter** [comment]
@silene Let me know if it looks good to you and I will merge

**Review by silene — APPROVED** [review]

**Comment by Alizter** [comment]
@coqbot merge now

**Pull request merged**